_layouts/repos.html

```diff
@@ -24,26 +24,39 @@
             <table class="table table-condensed table-striped table-hover">
               <thead>
                 <tr>
-                  <th style="text-align: center;"><span class="glyphicon glyphicon-th"></span></th>
+                  <th style="text-align: center;">
+                    <span class="glyphicon glyphicon-th"
+                          title="Package instance">
```

Comment:
Suggestion: change to "Repository instance**s**", since the icons below represent the number of instances for that repo.

Reply:
Agree, thanks for the suggestion! Done.